From 52f65c0868cb665a3212b884398642f842318caa Mon Sep 17 00:00:00 2001
From: "gaurav.mahajan" <gaurav.mahajan@lge.com>
Date: Mon, 22 May 2023 15:47:15 +0530
Subject: [PATCH] Change ownership of media directory to support non-root media
 services

Make user and group of "/media" directory as "media" so that
"/media" directory will be accessed by non-root media services

Upstream-Status: Inappropriate [webOS specific]
---
 tmpfiles.d/systemd.conf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tmpfiles.d/systemd.conf.in b/tmpfiles.d/systemd.conf.in
index 11a45a3f4b..a1fa787c1d 100644
--- a/tmpfiles.d/systemd.conf.in
+++ b/tmpfiles.d/systemd.conf.in
@@ -68,6 +68,7 @@ d /var/lib/systemd/ephemeral-trees 0755 root root 0
 d /var/lib/private 0700 root root -
 d /var/log/private 0700 root root -
 d /var/cache/private 0700 root root -
+d /media 0775 media media -
 
 {% if ENABLE_EFI %}
 # Copy sd-stub provided PCR signature and public key file from initrd into /run/, so that it will survive the initrd stage
